main.py

```diff
@@ -5,103 +5,135 @@
 
 
 app = Flask(__name__)
-app.config.from_object('config.Config')
+app.config.from_object("config.Config")
+
+from models import HadithCollection, Book, Chapter, Hadith
 
-from models import db, HadithCollection, Book, Chapter, Hadith
 
 @app.before_request
 def verify_secret():
-    if not app.debug and request.headers.get('x-aws-secret') != app.config['AWS_SECRET']:
+    if (not app.debug and request.headers.get("x-aws-secret") != app.config["AWS_SECRET"]):
         abort(401)
 
+
 @app.errorhandler(HTTPException)
 def jsonify_http_error(error):
-    response = {
-        'error': {
-            'details': error.description,
-            'code': error.code
-        }
-    }
+    response = {"error": {"details": error.description, "code": error.code}}
 
     return jsonify(response), error.code
 
+
 def paginate_results(f):
     @functools.wraps(f)
     def decorated_function(*args, **kwargs):
-        limit = int(request.args.get('limit', 50))
-        page = int(request.args.get('page', 1))
+        limit = int(request.args.get("limit", 50))
+        page = int(request.args.get("page", 1))
 
-        queryset = f(*args, **kwargs).paginate(page=page, per_page=limit, max_per_page=100)
+        queryset = f(*args, **kwargs).paginate(
+            page=page, per_page=limit, max_per_page=100
+        )
         result = {
-            'data': [x.serialize() for x in queryset.items],
-            'total': queryset.total,
-            'limit': queryset.per_page,
-            'previous': queryset.prev_num,
-            'next': queryset.next_num
+            "data": [x.serialize() for x in queryset.items],
+            "total": queryset.total,
+            "limit": queryset.per_page,
+            "previous": queryset.prev_num,
+            "next": queryset.next_num,
         }
         return jsonify(result)
+
     return decorated_function
 
+
 def single_resource(f):
     @functools.wraps(f)
     def decorated_function(*args, **kwargs):
         result = f(*args, **kwargs).first_or_404()
         result = result.serialize()
         return jsonify(result)
+
     return decorated_function
 
-@app.route('/', methods=['GET'])
+
+@app.route("/", methods=["GET"])
 def home():
     return "<h1>Welcome to sunnah.com API.</p>"
```

Comment:
Can we fix this in this commit as well? Not really python issue but it's a lint error nonetheless :)

Reply:
Sorry i was just confused which line you were referring to? This just adds a new line after `return decorated_function` ?

Reply:
Sorry I meant the html part which has mismatched tags.

Reply:
Oh sorry i overlooked that haha! Fixed it :)